src/resolver/resolver.zig

```diff
@@ -2881,14 +2888,23 @@ pub const Resolver = struct {
 
         const esm_resolution = esmodule.resolveImports(import_path, imports_map.root);
 
-        if (esm_resolution.status == .PackageResolve)
+        if (esm_resolution.status == .PackageResolve) {
+            if (JSC.HardcodedModule.Aliases.get(esm_resolution.path, .bun)) |builtin| {
```

Comment:
This needs to only apply when using Bun. This would do weird things in the browser